hs_file_types/models/timeseries.py

```diff
@@ -1357,6 +1373,12 @@ def extract_cv_metadata_from_blank_sqlite_file(target):
         shutil.rmtree(os.path.dirname(temp_csv_file))
 
 
+def _get_timestamped_file_name(file_name):
+    name, ext = os.path.splitext(file_name)
+    timestr = time.strftime("%Y%m%d-%H%M%S")
+    return "{}_{}{}".format(name, timestr, ext)
```

Comment:
Should ```.``` be added between timestr and ext?

Reply:
The `ext `variable already has the `.`

Reply:
OK. Got it. See if my follow-up suggestion for another comment makes sense. After that is addressed, I can approve this PR.